src/poliastro/tests/tests_twobody/test_orbit.py

```diff
@@ -589,3 +589,37 @@ def test_pqw_returns_dimensionless():
     assert p.unit == u.one
     assert q.unit == u.one
     assert w.unit == u.one
+
+
+def test_orbit_from_skycoord_inputs():
+
+    pos = [30000, 0, 0] * u.km
+    cartrep = CartesianRepresentation(*pos)
+
+    # Method fails if coordinate is not passed as SkyCoord instance
+    with pytest.raises(ValueError) as excinfo:
```

Comment:
And if we remove the `isinstance` check we don't need this part of the test either.

Reply:
Removed.